client/src/components/graph/graph.tsx

```diff
@@ -300,12 +285,17 @@ class Graph extends React.Component<GraphProps, GraphState> {
       },
       updateOverlay: false,
       testImageSrc: null,
+      isDeepZoomSourceValid: true,
```

Comment:
this state is used to capture the state that when Openseadragon attempts to fetch the deep zoom manifest and failed either because it doesn't exist or whatever reason, then we set the state to `false` and hide the Openseadragon layer